server/pkg/schema/field_select.go

```diff
@@ -0,0 +1,51 @@
+package schema
+
+import "errors"
+
+var TypeSelect Type = "select"
+
+var (
+	ErrFieldValues       = errors.New("invalid values")
+	ErrFieldDefaultValue = errors.New("invalid default values")
+)
+
+type FieldSelect struct {
+	values       []string
+	defaultValue *int
+}
+
+// NewFieldTag
+// TODO: check if its ok to remove this
+func NewFieldSelect() *FieldSelect {
+	return &FieldSelect{
+		values:       nil,
+		defaultValue: nil,
+	}
+}
+
+func FieldSelectFrom(values []string, defaultValue *int) (*FieldSelect, error) {
+	if values == nil || len(values) == 0 {
```

Comment:
```suggestion
	if len(values) == 0 {
```

`len([]T(nil)` is always 0